python/lsst/cp/verify/verifyCalib.py

```diff
@@ -236,3 +278,57 @@ def verify(self, inputCalib, statisticsDict, camera=None, exposure=None):
             subclass.
         """
         raise NotImplementedError("Subclasses must implement verification criteria.")
+
+    def repackStats(self, statisticsDict, dimensions):
+        """Repack information into flat tables.
+
+        This method should be redefined in subclasses.
```

Comment:
If this should be redefined, why isn't it an ABC with no code?

Reply:
Another dropped docstring.  This is a basic repacker for the simplest cases, and more complicated cases will need to replace it.  Docstring updated here and in `verifyStats.py` as well.